src/features/tag/components/TagMananger.tsx

```diff
@@ -61,7 +61,7 @@ const TagManager = ({
             setSelectedTags={setSelectedTags}
           />
           <CommandList
-            className={availableTags.length > 0 ? "mt-2" : undefined}
+            className={`overflow-auto ${availableTags.length > 0 ? "mt-2" : ""}`}
```

Comment:
This is the change for the long tags, correct?

Reply:
Yes 🙂